src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -42,27 +42,30 @@ public double getValue() {
     }
 
     /**
-     * Fetch a {@link Association} by its value.
+     * Fetch a {@link Association} by its exact value.
      *
      * @param value The friendliness value
      * @return The association related to the friendliness value. <code>null</code> if there is no {@link Association} with the provided value.
      * @author 086
      */
-    public static Association byValue(double value) {
+    public static Association byExactValue(final double value) {
         return Arrays.stream(values()).filter(association -> association.getValue() == value).findAny().orElseGet(null);
     }
 
     /**
-     * Fetch the {@link Association} closest to a friendliness value.
-     * <p>Edge cases: <code>0.5</code> becomes {@link Association#ALLY} and <code>-0.5</code> becomes {@link Association#ENEMY}</p>
+     * Fetch a {@link Association} by what the value means.
+     * If the provided value is lower than the standardised neutral value, this method will return {@link Association#ENEMY}
+     * If the provided value is higher than the standardised neutral value, this method will return {@link Association#ALLY}
+     * If the provided value is equal to the standardised neutral value, this method will return {@link Association#NEUTRAL}
```

Comment:
I'd format this as a HTML list.
IIRC, it's:
```html
<ul>
    <li>Some item</li>
    <li>Another item</li>
</ul>
```
which produces:
<ul>
    <li>Some item</li>
    <li>Another item</li>
</ul>